lib/v2/datasource/remote/api/send_eos_transaction_repository.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:async/async.dart';
+import 'package:eosdart/eosdart.dart';
+import 'package:seeds/v2/datasource/remote/api/eos_repository.dart';
+
+export 'package:async/src/result/result.dart';
+
+class SendTransactionRepository extends EosRepository {
```

Comment:
This is the function was in the EosRepository right? or is it something else?

Reply:
it was